testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestCsvFileHiveTable.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.hive;
+
+import io.trino.tempto.ProductTest;
+import org.testng.annotations.Test;
+
+import static io.trino.tests.product.utils.QueryExecutors.onHive;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class TestCsvFileHiveTable
+        extends ProductTest
```

Comment:
Do you need product tests for this? It sounds heavy for such detailed test. See `io.trino.plugin.hive.TestHiveConnectorTest#testCreateTableWithHeaderAndFooter`.

Reply:
I think it is beneficial to have it. To verify Hive can read the data we created (no we actually allow writing data, not just throw exception). But maybe `testCreateTextFileTableAsSelectSkipHeaderFooter` will be enough.

Surely modify/extend tests @kokosing pointed to.

Reply:
@kokosing please correct me if I'm wrong but I don't see the way to actually test if data is written the way we want in `io.trino.plugin.hive.TestHiveConnectorTest`. The infrastructure there seems not to even be able to parse CTAS so unfortunately I don't see the way to utilize it without very heavy lifting. 

@losipiuk `testCreateTextFileTableAsSelectSkipHeaderFooter` would be enough for TEXTFILES. But this change is for both TEXT and CSV files this is why I added new test class. If you think that testing only TEXTFILES or testing CSV in `TestTextFileHiveTable`  is enough then I'm also fine with it.

Reply:
> The infrastructure there seems not to even be able to parse CTAS 

I don't understand. I see CTAS used here `io.trino.plugin.hive.TestHiveConnectorTest#testIoExplain` so it should work.

> actually test if data is written the way we want

Why? You can get a `SELECT $location FROM <table>` and you can even read or write files manually. What else is missing?

Reply:
> @losipiuk testCreateTextFileTableAsSelectSkipHeaderFooter would be enough for TEXTFILES. But this change is for both TEXT and CSV files this is why I added new test class. If you think that testing only TEXTFILES or testing CSV in TestTextFileHiveTable is enough then I'm also fine with it.

I am fine with leaving both PTs.

Reply:
@kokosing sorry my mistake, I can't write a proper CTAS ... I also added tests to the class you pointed. Thanks!